### ISSUE TITLE
Vertical slabs kill grass beneath

### ISSUE THREAD

**Issue by Heptagon-ru** [anegostudios/VintageStory-Issues#200]
Verticsal slabs kill grass beneath them, which may be unwanted for some decorative builds.
Please consider changing that.
```
And just adding this code block in a futile attempt to memorise copygirl's advice how to copy crash reports
```
![2018-10-02_14-01-00](https://user-images.githubusercontent.com/25122345/46344995-9d609700-c64b-11e8-8049-04dd4360b58e.png)

**Comment by tyronx** [comment]
This might be fixed in 1.7.5, but you need to break and place the slab again

**Comment by Heptagon-ru** [comment]
Nope, still kills grass under freshly placed slabs. 1.7.5, pub test server.
![2018-10-16_18-57-01](https://user-images.githubusercontent.com/25122345/47029985-6de67a00-d175-11e8-80c6-0082d70e4bff.png)

**Comment by tyronx** [comment]
does this still happen in v1.7.8 or later versions?

**Comment by Heptagon-ru** [comment]
Yes. 1.8 pre 7, official server
![2018-12-13_21-42-49](https://user-images.githubusercontent.com/25122345/49959991-2172ad00-ff20-11e8-830a-141ab79372bd.png)

**Comment by Bloodsaw** [comment]
This is fixed, isnt it?
![image](https://user-images.githubusercontent.com/61588797/75725233-51060480-5ce0-11ea-8d1b-ec996b948d74.png)

**Comment by tyronx** [comment]
Yes, I think so. On your screenshot your slabs don't seem to touch the ground though

**Comment by Bloodsaw** [comment]
Oopsie, but yes I think its fixed:

![image](https://user-images.githubusercontent.com/61588797/75792443-47c37900-5d6e-11ea-8eaf-b53764b397d7.png)

The higher gras gets killed, but the green fur on blocks persist. 
I think this is the expected behavior?

**Comment by tyronx** [comment]
yes, that's expected

**Comment by Bloodsaw** [comment]
Ok maybe only partly resolved. Loaded the savegame today: 
![image](https://user-images.githubusercontent.com/61588797/75914213-aa8f4000-5e54-11ea-8579-1eb71c093440.png)
The bug doesn't occur upon placing, but on reloading the chunks I guess.

**Comment by Craluminum2413** [comment]
1.15.0-pre.3
![2021-06-19_00-05-03](https://user-images.githubusercontent.com/69315569/122615931-13beb500-d092-11eb-8f6e-b52ecdc27729.png)

**Comment by noelle-lavenza** [comment]
Is it possible they're blocking the light? It looks like `ChunkIlluminator` doesn't take into account side-based opacity, just the `lightAbsorption` value, which I believe is unset for slabs and therefore defaults to 99. That way, even though it's set to be transparent and non-solid, it still blocks the light enough for the grass to start dying.

**Comment by Craluminum2413** [comment]
Can't reproduce in 1.21.0-pre.2

**Comment by noelle-lavenza** [comment]
> Can't reproduce in 1.21.0-pre.2
> <img alt="Image" width="495" height="621" src="https://private-user-images.githubusercontent.com/69315569/465635522-1c9470ac-a99a-4c5f-8be1-64e3984adf7b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM1ODcyMTMsIm5iZiI6MTc1MzU4NjkxMywicGF0aCI6Ii82OTMxNTU2OS80NjU2MzU1MjItMWM5NDcwYWMtYTk5YS00YzVmLThiZTEtNjRlMzk4NGFkZjdiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzI3VDAzMjgzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzNDI0NjJjMTE3MzMyYTIxZDhiZTI2MGZjZGI0N2YzZTdkZjYwYmYzZTQ0Y2Q1NzlkZDhiM2I5Y2IyNzNiMTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.quX1DjC5LRFCdAL1yvjfyLWyeuGm-krG6eyuAen9XgA">

I don't see how this screenshot demonstrates that it couldn't be reproduced, there's clearly dead grass beneath those slabs. Or are you demonstrating that it grew from dirt->grass instead?

**Comment by Craluminum2413** [comment]
I didn't test if it happens on chunk reloading
